adagrams/game.py

```diff
@@ -1,13 +1,197 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
+score_chart = {'A':1,
+                   'B':3,
+                   'C':3,
+                   'D':2,
+                   'E':1,
+                   'F':4,
+                   'G':2,
+                   'H':4,
+                   'I':1,
+                   'J':8,
+                   'K':5,
+                   'L':1,
+                   'M':3,
+                   'N':1,
+                   'O':1,
+                   'P':3,
+                   'Q':10,
+                   'R':1,
+                   'S':1,
+                   'T':1,
+                   'U':1,
+                   'V':4,
+                   'W':4,
+                   'X':8,
+                   'Y':4,
+                   'Z':10
+                   }
+
+
+def make_dict(array):
+    '''
+    Creates a dictionary of letters in the array as keys, and frequency of letters as values 
+    input: a sting or a list of characters
+    output: a dictionary of letters as keys, and frequency as values
+    '''
+    make_dict = {}
+    for character in array:
+        if character not in make_dict:
+            make_dict[character] = 1
+        else:
+            make_dict[character] += 1
+    return make_dict
+
+
 def draw_letters():
-    pass
+    # Convert letter pool dictionary to a list with repeated letters 
+    # the list above has 98 letters.
+    # make a random selection out of 98 letters of pool list.
+    # add the selected letter to the hand and add a counter that shows frequency of the letter in the hand
+    # draw hand should be a dict array with k/v as letters / freq
+    # check if freq of the selected letter is less than the freq of the pool
+    # repeat this 10 times!
+    # return a list of hand letters!
+    
+    letter_pool_list = []
+    hand_dict = {}
+    hand_list = []
+
+    for letter, frequency in LETTER_POOL.items():
+        for i in range(frequency):
+            letter_pool_list.append(letter)
+
+    # for i in range(10): Can't use for loop in indetermined number of loops
+    hand_counter = 1
```

Comment:
We already have an easy way of keeping track of how many cards are in our hand (`len(hand_list)`), so adding in a hand_counter essentially adds duplicate data into our code. In cases like these, I would err on the side of using length where you can instead of a counter!